ci-operator/config/maistra/api/maistra-api-maistra-2.1.yaml

```diff
@@ -0,0 +1,36 @@
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.1"
```

Comment:
Since you are creating jobs for 2.1, 2.2 and 2.3, I'd change the title of this PR.